EBA/Taschenrechner/Grundrechner/GrundRechnerTest/Geomitry.cs

```diff
@@ -26,7 +26,7 @@ public void Testcos()
         [Fact]
         public void radtest()
         {
-            Assert.True(new TestGeo().CircleRadus(120) == 6.1803872323710367);
+            Assert.True(new TestGeo().CircleRadus(120) == 6.180387232371034);
```

Comment:
????
That's the right is value:
6,1803872323710332854750837649838

The the right number here must have a `3` at the end. not a `4`

Calculation:

√(120 / π)

Reply:
Nur wenn deine Pi mehr Ziffern hat als unsere. :)
die Genauigkeit hängt davon ab, wie gut Ihr Kalkulator die x-te Stelle hinter dem Komma lösen kann.